components/eam/src/physics/cam/microp_aero.F90

```diff
@@ -749,124 +612,19 @@ subroutine microp_aero_run ( &
-
-               ! get droplet activation rate
-
-               call ndrop_bam_run( &
-                  wsub(i,k), t(i,k), rho(i,k), naer2(i,k,:), naer_all, &
-                  naer_all, maerosol(i,k,:),  &
-                  dum2)
-               dum = dum2
-            else
-               dum = 0._r8
-            end if
-
-            npccn(i,k) = (dum*lcldm(i,k) - nc(i,k))/deltatin
-         end do
-      end do
-      call t_stopf('droplet_act_bulk_aero')
-
-   end if
 
 
    !cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
    ! Contact freezing  (-40<T<-3 C) (Young, 1974) with hooks into simulated dust
    ! estimate rndst and nanco for 4 dust bins here to pass to MG microphysics
-
-   do k = top_lev, pver
-      do i = 1, ncol
-
-         if (t(i,k) < 269.15_r8) then
-
-            if (clim_modal_aero) then
-
-               ! For modal aerosols:
-               !  use size '3' for dust coarse mode...
-               !  scale by dust fraction in coarse mode
-               
-               dmc  = coarse_dust(i,k)
-               ssmc = coarse_nacl(i,k)
-
-               if ( separate_dust ) then
-                  ! 7-mode -- has separate dust and seasalt mode types and no need for weighting 
-                  wght = 1._r8
-               else
-                  ! 3-mode -- needs weighting for dust since dust and seasalt are combined in the "coarse" mode type
-                  wght = dmc/(ssmc + dmc)
-               endif
-
-               if (dmc > 0.0_r8) then
-                  nacon(i,k,3) = wght*num_coarse(i,k)*rho(i,k)
-               else
-                  nacon(i,k,3) = 0._r8
-               end if
-
-               !also redefine parameters based on size...
-
-               rndst(i,k,3) = 0.5_r8*dgnumwet(i,k,mode_coarse_dst_idx)
-               if (rndst(i,k,3) <= 0._r8) then 
-                  rndst(i,k,3) = rn_dst3
-               end if
-
-            else
-
-               !For Bulk Aerosols: set equal to aerosol number for dust for bins 2-4 (bin 1=0)
-
-               if (idxdst2 > 0) then 
-                  nacon(i,k,2) = naer2(i,k,idxdst2)
-               end if
-               if (idxdst3 > 0) then 
-                  nacon(i,k,3) = naer2(i,k,idxdst3)
-               end if
-               if (idxdst4 > 0) then 
-                  nacon(i,k,4) = naer2(i,k,idxdst4)
-               end if
-            end if
-
-         end if
-      end do
-   end do
-
-   !cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
-   !bulk aerosol ccn concentration (modal does it in ndrop, from dropmixnuc)
-
-   if (.not. clim_modal_aero) then
-
-      ! ccn concentration as diagnostic
-      call t_startf('ndrop_bam_ccn')
-      call ndrop_bam_ccn(lchnk, ncol, maerosol, naer2)
-      call t_stopf('ndrop_bam_ccn')
-
-      deallocate( &
-         naer2,    &
-         maerosol)
-
-   end if
+   ! removed because the default contact freezing scheme is now CNT
```

Comment:
This comment might not make sense to a newcomer since the old code has been removed.

Reply:
removed the comment